wire-ios-data-model/Source/Notifications/ObjectObserverTokens/UserChangeObserver.swift

```diff
@@ -16,11 +16,7 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
-
-@objc(ZMUserChangeObserver)
-public protocol UserChangeObserver: NSObjectProtocol {
+@objc(ZMUserObserver)
+public protocol UserObserver {
```

Comment:
```suggestion
public protocol UserObserving {
```
as this is a capability than a noun.

Reply:
Good point, I changed it. 👍